src/config.rs

```diff
@@ -54,8 +54,8 @@ pub static Z_CONFIG_SCOUTING_TIMEOUT_KEY: &c_char =
 pub static Z_CONFIG_SCOUTING_DELAY_KEY: &c_char =
     unsafe { &*(b"scouting/delay\0".as_ptr() as *const c_char) };
 #[no_mangle]
-pub static Z_CONFIG_ADD_TIMESTAMP_KEY: &c_char =
-    unsafe { &*(b"add_timestamp\0".as_ptr() as *const c_char) };
+pub static Z_CONFIG_TIMESTAMPING_ENABLED_KEY: &c_char =
```

Comment:
I see now that the value is also different, and makes more sense. That doesn't need to impact the symbol's name though..